### PR TITLE
Resprites Pod Secondary Systems

### PR DESCRIPTION
**Pull request by gunwithaman**
[Vehicles]
[Sprites]

## About the PR 
Resprites most of the space pod secondary systems. 

## Why's this needed? 
All the secondary systems shared the same sprite, making it difficult to make out from a glance what each one did. These new sprites hopefully will create some variety while being vaguely similar enough to visually tell players that they all go in the same slot. Was on the wanted respites list.

secondary (1)

from left to right: Cargo Hold, Ore Scoop/Hold, Tractor Beam, Ship Navigational GPS, Syndicate Explosive Entry Device, Medusa Stealth System 300

first-time doing this
## Changelog 

```changelog
(u)Zaniff
(+)Resprited the pod secondary systems.
```

### CODE REVIEW

**Comment by blackrep** [comment]
Don't need to put the letter in front of the tags. Just do [Vehicles]. Also the sprites tag is automatically added to any PR that touches a `.dmi` file.

**Review by flappybatpal — APPROVED** [review]

**Review by Gannets — APPROVED** [review]
I like the look of these!

File: code/modules/transport/pods/secondary_system.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 	hud_state = "cloak"
 	f_active = 1
 	var/image/shield = null
+	icon_state= "medusa"
```
Comment (ZeWaka) [inline code/modules/transport/pods/secondary_system.dm:27]:
```suggestion
	icon_state = "medusa"
```

File: code/modules/transport/pods/secondary_system.dm [hunk 2]
```diff
@@ -67,6 +68,7 @@
 	var/capacity = 300
 	hud_state = "cargo"
 	f_active = 1
+	icon_state= "ore_hold"
```
Comment (ZeWaka) [inline code/modules/transport/pods/secondary_system.dm:71]:
```suggestion
	icon_state = "ore_hold"
```

File: code/modules/transport/pods/secondary_system.dm [hunk 3]
```diff
@@ -322,6 +324,7 @@
 	f_active = 1
 	power_used = 80
 	hud_state = "tractor_beam"
+	icon_state= "trac_beam"
```
Comment (ZeWaka) [inline code/modules/transport/pods/secondary_system.dm:327]:
```suggestion
	icon_state = "trac_beam"
```

File: code/modules/transport/pods/secondary_system.dm [hunk 4]
```diff
@@ -466,6 +469,7 @@
 	desc = "A useful navigation device for those lost in space."
 	f_active = 1
 	power_used = 50
+	icon_state= "ship_gps"
```
Comment (ZeWaka) [inline code/modules/transport/pods/secondary_system.dm:472]:
```suggestion
	icon_state = "ship_gps"
```

File: code/modules/transport/pods/secondary_system.dm [hunk 5]
```diff
@@ -789,6 +793,7 @@
 	var/crashhits = 10
 	var/in_bump = 0
 	hud_state = "seed"
+	icon_state= "pod_seed"
```
Comment (ZeWaka) [inline code/modules/transport/pods/secondary_system.dm:796]:
```suggestion
	icon_state = "pod_seed"
```

**Review by ZeWaka — APPROVED** [review]
looks good, thanks for the pr!

**Pull request merged**